src/main/scala/com/github/timgent/dataflare/metrics/MetricDescriptor.scala

```diff
@@ -79,6 +74,45 @@ object MetricDescriptor {
     override type MC = SumValuesMetricCalculator[MV]
   }
 
+  /**
+    * A metric that calculates the min value of rows in your dataset
+    * @param onColumn
+    * @param filter filter to be applied before the size is calculated
+    * @param
+    * @tparam MV
+    */
+
+  case class MinValueMetric[MV <: NumericMetricValue: MetricValueConstructor](
```

Comment:
Pls add docstring

Reply:
docstring was already present but due to newline is appeared that no docstrings are there. Kindly check once more